### ISSUE TITLE
Cys1 MGI:2177632  not involved in regulation of transcription

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-annotation#3167]
Hi @hdrabkin 

PMID:24349431 does not provide any convincing evidence that Cys1 (MGI:2177632) is involved in transcription - either MF or BP. The eveidence is really weak, there is no DNA binding domain, it cooperates with a protein that has not been shown to be a DNA binding protein either. So, we suggest you remove: 

*  Cys1 | enables | GO:0000978 | RNA polymerase II cis-regulatory region sequence-specific DNA binding | ECO:0000314(IDA) | ECO:0000314 | (IDA) |   | PMID:24349431 |  
* Cys1 | enables | GO:0001228 | DNA-binding transcription activator activity, RNA polymerase II-specific | ECO:0000314(IDA) | ECO:0000314 | (IDA) |   | PMID:24349431 |   |   | has_input(MGI:MGI:97250),oc... |  
* Cys1 | enables | GO:0003714 | transcription corepressor activity | ECO:0000314(IDA) | ECO:0000314 | (IDA) |   | PMID:24349431 |   |   |  
* Cys1 | acts_upstream_of_or_within | GO:0043433 | negative regulation of DNA-binding transcription factor activity | ECO:0000314(IDA) | ECO:0000314 | (IDA) |   | PMID:24349431 |  
* Cys1 | acts_upstream_of_or_within | GO:0045944 | positive regulation of transcription by RNA polymerase II | ECO:0000314(IDA) | ECO:0000314 | (IDA) |   | PMID:24349431

You can keep the binding and the localization annotations.

Thanks, Pascale

**Comment by krchristie** [comment]
@pgaudet - Both @LiNiMGI and I read this paper in detail and discussed it together. We disagree with you that this paper does not provide any convincing evidence that Cys1 is involved in transcription. We feel this paper provides quite convincing evidence of the involvement of Cys1 in transcription. 

Please provide specific details about why you think this paper is not convincing, especially with respect to the EMSA.

Thanks,

-Karen

**Comment by colinlog** [comment]
@krchristie @hdrabkin @RLovering @pgaudet 

(i) The EMSAs are conducted with nuclear extracts that can therefore not distinguish direct binding from indirect (mediated by another protein with a DNA binding domain).  
(ii) There is a sequence specificity control missing in the EMSA, the competition is only done with unlabeled probe with the GN box. To demonstrate specificity, unlabeled probe without GN box should be shown to compete much less efficiently.

Therefore the EMSA experiment does not convincingly demonstrate direct DNA binding. We do not contest implication in transcription per se, but we do contest the sequence-specific DNA binding transcription factor activity annotations (GO:0000978 and GO:0001228).

Finally, NDN (NECDIN) is a member of the Panther protein family PTHR11736 most of which are known ubiquitin ligase associated proteins (eg MAGEA2 -- MAGEL2, ) that influence transcription but do not have demonstrated sequence-specific DNA binding domains, although non-nuclear members such as TRO also belong to this PTHR family.

In summary we argue to remove the sequence specific DNA binding annotations GO:0000978 and GO:0001228

We think the following annotations are fine:

Cys1 | enables | GO:0003714 | transcription corepressor activity | ECO:0000314(IDA) | ECO:0000314 | (IDA) |   | PMID:24349431 |   |   |  
Cys1 | acts_upstream_of_or_within | GO:0043433 | negative regulation of DNA-binding transcription factor activity | ECO:0000314(IDA) | ECO:0000314 | (IDA) |   | PMID:24349431 |  
Cys1 | acts_upstream_of_or_within | GO:0045944 | positive regulation of transcription by RNA polymerase II | ECO:0000314(IDA) | ECO:0000314 | (IDA) |   | PMID:24349431

Note that "acts_upstream_of_or_within" could be replaced by "involved_in"

**Comment by krchristie** [comment]
I have removed these 2 annotations:
- Cys1 | enables | GO:0000978 | RNA polymerase II cis-regulatory region sequence-specific DNA binding | ECO:0000314(IDA) | ECO:0000314 | (IDA) |   | PMID:24349431 |  
- Cys1 | enables | GO:0001228 | DNA-binding transcription activator activity, RNA polymerase II-specific | ECO:0000314(IDA) | ECO:0000314 | (IDA) |   | PMID:24349431 |   |   | has_input(MGI:MGI:97250),oc... |  

and kept the rest.
